ee/clickhouse/views/test/funnel/__snapshots__/test_clickhouse_funnel_person.ambr

```diff
@@ -0,0 +1,123 @@
+                              if(event = 'step two', 1, 0) as step_1,
+                              if(step_1 = 1, timestamp, null) as latest_1,
+                              if(event = 'step three', 1, 0) as step_2,
+                              if(step_2 = 1, timestamp, null) as latest_2
+                       FROM
+                         (SELECT e.event as event,
+                                 e.team_id as team_id,
+                                 e.distinct_id as distinct_id,
+                                 e.timestamp as timestamp,
+                                 e."$group_0" as aggregation_target,
+                                 e."properties" as "properties",
+                                 e."$group_0" as "$group_0",
+                                 pdi.person_id as person_id,
+                                 person.person_props as person_props,
+                                 groups_0.group_properties_0 as group_properties_0
+                          FROM events e
+                          INNER JOIN
+                            (SELECT distinct_id,
+                                    argMax(person_id, version) as person_id
+                             FROM person_distinct_id2
+                             WHERE team_id = 2
+                             GROUP BY distinct_id
+                             HAVING argMax(is_deleted, version) = 0) AS pdi ON e.distinct_id = pdi.distinct_id
+                          INNER JOIN
+                            (SELECT id,
+                                    argMax(properties, version) as person_props
+                             FROM person
+                             WHERE team_id = 2
+                             GROUP BY id
+                             HAVING max(is_deleted) = 0) person ON person.id = pdi.person_id
+                          INNER JOIN
+                            (SELECT group_key,
+                                    argMax(group_properties, _timestamp) AS group_properties_0
+                             FROM groups
+                             WHERE team_id = 2
+                               AND group_type_index = 0
+                             GROUP BY group_key) groups_0 ON "$group_0" == groups_0.group_key
+                          WHERE team_id = 2
+                            AND event IN ['step one', 'step three', 'step two']
+                            AND toDateTime(timestamp, 'UTC') >= toDateTime('2021-05-01 00:00:00', 'UTC')
+                            AND toDateTime(timestamp, 'UTC') <= toDateTime('2021-05-10 23:59:59', 'UTC')
+                            AND ((replaceRegexpAll(JSONExtractRaw(person_props, 'email'), '^"|"$', '') ILIKE '%g0%'
+                                  OR replaceRegexpAll(JSONExtractRaw(person_props, 'name'), '^"|"$', '') ILIKE '%g0%'
+                                  OR replaceRegexpAll(JSONExtractRaw(e.properties, 'distinct_id'), '^"|"$', '') ILIKE '%g0%'
```

Comment:
note that I've kept persons search alongside groups - maybe a decision to revisit but this feels like it could be useful